### PR TITLE
Stop coercing strings to integer and boolean

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Fixees #80

## Was the documentation (README, wiki) updated?
n/a

### CODE REVIEW

File: spec/cocina/models/description_spec.rb [hunk 1]
```diff
@@ -3,26 +3,5 @@
...
-        ]
-      }
-    end
-
-    it 'coerces to boolean' do
-      expect(instance.title.first.primary).to eq true
-      expect(instance.title.last.primary).to eq false
-    end
-  end
+  it 'should have tests', skip: true
```
[the remark this answers is not in the document]
Comment (jcoyne) [reply spec/cocina/models/description_spec.rb]: I don't think any of these really need tests right now, but rubocop complains about empty spec files.
